README.md

```diff
@@ -253,6 +253,7 @@ Code Vectors: Understanding Programs Through Embedded Abstracted Symbolic Traces
 * <img src="badges/10-pages-gray.svg" alt="10-pages" align="top"> [A Deep Learning Approach to Program Similarity](https://dl.acm.org/citation.cfm?id=3243131) - Niccolò Marastoni, Roberto Giacobazzi and Mila Dalla Preda, MASES 2018.
 * <img src="badges/6-pages-gray.svg" alt="6-pages" align="top"> [Recurrent Neural Network for Code Clone Detection](https://seim-conf.org/media/materials/2018/proceedings/SEIM-2018_Short_Papers.pdf#page=48) - Arseny Zorin and Vladimir Itsykson, SEIM 2018.
 * <img src="badges/8-pages-gray.svg" alt="8-pages" align="top"> [The Adverse Effects of Code Duplication in Machine Learning Models of Code](https://arxiv.org/abs/1812.06469) - Miltiadis Allamanis, 2018.
+* <img src="badges/10-pages-gray.svg" alt="10-pages" align="top"> [Learning-Based Recursive Aggregation of Abstract Syntax Trees for Code Clone Detection](https://ieeexplore.ieee.org/document/8668039) - Lutz Büch and Artur Andrzejak, SANER 2019
```

Comment:
```suggestion
* <img src="badges/10-pages-gray.svg" alt="10-pages" align="top"> [Learning-Based Recursive Aggregation of Abstract Syntax Trees for Code Clone Detection](https://ieeexplore.ieee.org/document/8668039) - Lutz Büch and Artur Andrzejak, SANER 2019.
```

Always a good idea to keep formatting consistent 😉

Reply:
I can't download an article without registration.
Is it possible to upload it somewhere else?
I want all articles to be available without registration